Server/mods/deathmatch/logic/CAccountManager.cpp

```diff
@@ -530,7 +530,8 @@ bool CAccountManager::LogIn(CClient* pClient, CClient* pEchoClient, const char*
             pEchoClient->SendEcho(SString("login: Account for '%s' is already in use", szAccountName).c_str());
         return false;
     }
-    if (bCheckPassword && (!IsValidPassword(szPassword) || !pAccount->IsPassword(szPassword)))
```

Comment:
is the `IsValidPassword` not required anymore?

Reply:
it looks like this function is checked before passwords are used https://github.com/multitheftauto/mtasa-blue/commit/bec0300e5fd456eeed775c773af7d99372de52cf

Reply:
Have I deleted it?

Reply:
yes

Reply:
My bad... Must have overlooked something.

Reply:
test 123

Reply:
Probably a good idea